Add recipe for babashka

licht1stein opened the pull request:
### Brief summary of what the package does

Emacs interface for [babashka](https://babashka.org/) tasks. 

### Direct link to the package repository

https://github.com/licht1stein/babashka.el

### Your association with the package

Maintainer

### Relevant communications with the upstream package maintainer


### Checklist

- [x] The package is released under a [GPL-Compatible Free Software License](https://www.gnu.org/licenses/license-list.en.html#GPLCompatibleLicenses)
- [x] I've read [CONTRIBUTING.org](https://github.com/melpa/melpa/blob/master/CONTRIBUTING.org)
- [x] I've used the latest version of [package-lint](https://github.com/purcell/package-lint) to check for packaging issues, and addressed its feedback
- [x] My elisp byte-compiles cleanly
- [x] I've used `M-x checkdoc` to check the package's documentation strings
- [x] I've built and installed the package using the instructions in [CONTRIBUTING.org](https://github.com/melpa/melpa/blob/master/CONTRIBUTING.org)

riscy commented:
Thanks for your close adherence to the checklist.

I am looking at your call to `(async-shell-command command output-buffer)` and see that the command doesn't use `shell-quote-argument` anywhere.  I'm concerned this could be an issue, for example if the <task> part of `bb <task>` has shell control characters in it (like apostrophes, semicolons, that sort of thing).  (Is this a possibility?)

Minor:
- babashka.el#L83: No `format` required; `message` takes an f-string

licht1stein commented:
@riscy thank you for spotting that, I added args escaping in this commit. Also removed redundant format from L83. Thanks!

https://github.com/licht1stein/babashka.el/commit/173a274f092f162a439ee3d72baea7b9cdfd16c7

riscy commented:
Thank you!  Merging.
